common/source/docs/common-lua-scripts.rst

```diff
@@ -103,12 +103,18 @@ A Location userdata object supports the following calls:
 
 - :code:`loiter_xtrack([should_xtrack])` - If the location is used for a loiter location this flags if the aircraft should track from the center point, or from the exit location of the loiter.
 
-- :code:`get_distance(other_loc)` - Given a Location this calculates the horizontal distance between the two locations in meters.
+- :code:`get_distance(other Location)` - Given a Location this calculates the horizontal distance between the two locations in meters.
 
 - :code:`offset(offset_north, offset_east)` - Translates this Location by the specified number of meters.
 
 - :code:`get_vector_from_origin_NEU()` - Returns nil or Vector3f that contains the offset from the EKF origin to this location. This will return nil if the EKF origin wasn’t available at the time this was called.
 
+- :code:`get_bearing(other Location)` - Given a Location this calculates the relative bearing to the location in degrees
```

Comment:
No space, same comment. This is also a value in radians not degrees.

Reply:
fixed